src/wconfig/WorkerConfig.h

```diff
@@ -88,7 +88,11 @@ class ConfigValResultDeliveryProtocol : public util::ConfigVal {
 private:
     ConfigValResultDeliveryProtocol(std::string const& section, std::string const& name, bool required,
                                     std::string const& defVal, bool hidden)
-            : ConfigVal(section, name, required, hidden), _val(parse(defVal)) {}
+            : ConfigVal(section, name, required, hidden) {
+        _val = parse(defVal);
+        _defVal = _val;
+    }
+
```

Comment:
I ran ran into the same problem in the branch. My solution there is
```    
ConfigValResultDeliveryProtocol(std::string const& section, std::string const& name, bool required,
                                    std::string const& defVal, bool hidden)
            : ConfigVal(section, name, required, hidden), _defVal(parse(defVal)), _val(_defVal) {}
TEnum const _defVal;  ///< Default value for the item this class is storing.
TEnum _val;           ///< Value for the item this class is storing.
```

Reply:
That's the same as I have.